common/conf/conf.cpp

```diff
@@ -210,11 +210,15 @@ expected::ExpectedSize MenderConfig::ProcessCmdlineArgs(
 		}
 	}
 
-	auto ex_log_level = log::StringToLogLevel(log_level);
-	if (!ex_log_level) {
-		return expected::unexpected(ex_log_level.error());
+	SetLevel(log::kDefaultLogLevel);
+
+	if (log_level != "") {
+		auto ex_log_level = log::StringToLogLevel(log_level);
+		if (!ex_log_level) {
+			return expected::unexpected(ex_log_level.error());
```

Comment:
Hmm, is it better to just log a warning and go with the default?

Reply:
I think we should avoid silently accepting invalid options (well, silently in the sense that you have to actively check the log), unless there's a high risk of compromising robustness. I don't think that's the case here.